lib/prawn/font/ttc.rb

```diff
@@ -0,0 +1,35 @@
+# font.rb : The Prawn font class
+#
+# Copyright November 2008, Jamis Buck. All Rights Reserved.
+#
+# This is free software. Please see the LICENSE and COPYING files for details.
+#
```

Comment:
Ha, sorry... as you can see from the date, that was actually not recent. :) I copied the font/dfont.rb file (which I also contributed, more than eight years ago, apparently!) and adapted it to TTC, and forgot to clean up the header comment.  I'll clean it up there, and on dfont.rb.